addons/dotnet/dotnet_imagestreams.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
How do you plan to keep these files in sync with s2i-dotnetcore repo?

Reply:
We are planing to sync the imagestreams and templates every release of OpenShift. Do you have any other suggestion.

Reply:
I saw your comment about v3.9.0. So IMO this add-on is not required anymore.